releasenotes/notes/fix-1q-non-determinism-vf2-75490119547bf656.yaml

```diff
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Fixed an issue in the :class:`.VF2Layout` transpiler pass where even with
+    a fixed seed set the output from the pass was potentially non-deterministic;
+    specifically if the input circuit had any active qubits that only contained
+    single qubit operations.
```

Comment:
Technically I suppose the condition was that you had at least two individually isolated qubits with the same number of operations on them.

Reply:
Or the average error rates for any two free qubits are the same. You could have a single isolated qubit that would get placed randomly between those tied error rates. Both of those details felt a bit too specific for trying to explain in a release note though.